lightning_pose_app/ui/train_infer.py

```diff
@@ -1172,12 +1181,18 @@ def _render_streamlit_fn(state: AppState):
 
         eks_tab = st.container()
         with eks_tab:
-
             st.header("Create an ensemble of models")
+            # Create a model list with no pre exist eks models in it
+            model_dir = os.path.join(state.proj_dir[1:], MODELS_DIR)
+            trained_models_no_eks = find_models(
+                model_dir,
+                must_contain_predictions=False,
+                must_contain_config=True
+            )
             selected_models = st.multiselect(
-               "Select models for ensembling",
-               sorted(state.trained_models, reverse=True),
-               help="Select which models you want to create an new ensemble model",
+                "Select models for ensembling",
+                sorted(trained_models_no_eks, reverse=True),
```

Comment:
this isn't accessing `*state*.trained_models_no_eks` - does this work fine? if so, we can probably just remove the flow attribute `trained_models_no_eks`